Added definition for instance-side. (#262)
pharo-open-documentation/pharo-wiki: General/Glossary.md

## Instance

## Instance-side

> The instance-side of a meta-class refers to its class. This class contain methods that can be send to its instances.

Reviewer: ```suggestion
The instance-side of a meta-class refers to its class. This class contains methods that can be sent to its instances.
```

For example, `#x` method (which returns the x coordinate of a point) is held by the class `Point`.

> In Pharo, both `Class` and `MetaClass` understand `#instanceSide` method.

Reviewer: ```suggestion
In Pharo, both `Class` and `Metaclass` understand `#instanceSide` method.
```

For example:

```Smalltalk
Point instanceSide. "Point"
Point class instanceSide. "Point"
Point includesSelector: #x. "true"
Point class includesSelector: #x. "false"
```

The best way to understand what is the instance-side is to have a look at `#instanceSide` methods implementations:

```Smalltalk
Class>>#instanceSide
	^ self
```

```Smalltalk

> MetaClass>>#instanceSide

Reviewer: ```suggestion
Metaclass>>#instanceSide
```
